htmd/parameterization/fftype.py

```diff
@@ -90,3 +111,23 @@ def __init__(self, mol, method=FFTypeMethod.CGenFF_2b6, rtf=None, prm=None):
                 raise RuntimeError("FFTyping failed reading Match output: see {}".format(tmpdir))
         else:
             raise RuntimeError("Unknown method for FFType: {}".format(method))
+
+if __name__ == '__main__':
+
+    from htmd.parameterization.ffmolecule import FFMolecule
+
+    mol = FFMolecule('../data/building-protein-ligand/benzamidine.mol2')
+
+    dir = tempfile.mkdtemp()
+
+    ff = FFType(mol, method=FFTypeMethod.CGenFF_2b6)
+    ff._rtf.write(os.path.join(dir, 'test.rtf'))
+    ff._prm.write(os.path.join(dir, 'test.prm'))
+
+    ff = FFType(mol, method=FFTypeMethod.GAFF)
+    ff._prm.writeFrcmod(ff._rtf, os.path.join(dir, 'test.frcmod'))
```

Comment:
If the files are small you can add them to `htmd/data/test-fftype/` and then compare them so that the test actually test for consistent results